{"metadata":{"usedHelpers":[],"marked":[],"modules":{"imports":[],"exports":{"exported":[],"specifiers":[]}}},"options":{"filename":"/packages/practicalmeteor:mocha/test/integration/fixtures/hooks/before.hook.async.error.js","filenameRelative":"/packages/practicalmeteor:mocha/test/integration/fixtures/hooks/before.hook.async.error.js","env":{},"retainLines":false,"highlightCode":true,"suppressDeprecationMessages":false,"presets":[],"plugins":[[[],{"polyfill":false}],[[],null],[[],null],[[],null],[[],null],[[],null],[[],null],[[],null],[[],null],[[],null],[[],null],[[],null],[[],null],[[],null],[[],null],[[],null],[[],{"loose":true}],[[],{"loose":true}],[[],null],[[],{"loose":true}],[[],null],[[],null],[[],null],[[],null],[[],null],[[],null],[[],null],[[],{"loose":true}],[[],null],[[],null],[[],null],[[],null],[[],null],[[],null],[[],null],[[],null]],"ignore":[],"code":true,"metadata":true,"ast":false,"comments":true,"compact":false,"minified":false,"sourceMap":true,"sourceMaps":true,"sourceMapTarget":"/packages/practicalmeteor:mocha/test/integration/fixtures/hooks/before.hook.async.error.js.map","sourceFileName":"/packages/practicalmeteor:mocha/test/integration/fixtures/hooks/before.hook.async.error.js","babelrc":false,"sourceType":"module","moduleIds":false,"passPerPreset":false,"basename":"before.hook.async.error"},"ignored":false,"code":"describe('spec 1', function () {\n  before(function (done) {\n    console.log('before');\n    process.nextTick(function () {\n      throw new Error('before hook error');\n    });\n  });\n  it('should not be called because of error in before hook', function () {\n    console.log('test 1');\n  });\n  it('should not be called because of error in before hook', function () {\n    console.log('test 2');\n  });\n});\ndescribe('spec 2', function () {\n  it('should be called, because hook error was in a sibling suite', function () {\n    console.log('test 3');\n  });\n});","ast":null,"map":{"version":3,"sources":["/packages/practicalmeteor:mocha/test/integration/fixtures/hooks/before.hook.async.error.js"],"names":[],"mappings":"AAAA,SAAS,QAAT,EAAmB,YAAY;AAC7B,SAAO,UAAU,IAAV,EAAgB;AACrB,YAAQ,GAAR,CAAY,QAAZ;AACA,YAAQ,QAAR,CAAiB,YAAY;AAC3B,YAAM,IAAI,KAAJ,CAAU,mBAAV,CAAN;AACD,KAFD;AAGD,GALD;AAMA,KAAG,sDAAH,EAA2D,YAAY;AACrE,YAAQ,GAAR,CAAY,QAAZ;AACD,GAFD;AAGA,KAAG,sDAAH,EAA2D,YAAY;AACrE,YAAQ,GAAR,CAAY,QAAZ;AACD,GAFD;AAGD,CAbD;AAcA,SAAS,QAAT,EAAmB,YAAY;AAC7B,KAAG,6DAAH,EAAkE,YAAY;AAC5E,YAAQ,GAAR,CAAY,QAAZ;AACD,GAFD;AAGD,CAJD","file":"/packages/practicalmeteor:mocha/test/integration/fixtures/hooks/before.hook.async.error.js.map","sourcesContent":["describe('spec 1', function () {\n  before(function (done) {\n    console.log('before');\n    process.nextTick(function () {\n      throw new Error('before hook error');\n    });\n  });\n  it('should not be called because of error in before hook', function () {\n    console.log('test 1');\n  });\n  it('should not be called because of error in before hook', function () {\n    console.log('test 2');\n  });\n});\ndescribe('spec 2', function () {\n  it('should be called, because hook error was in a sibling suite', function () {\n    console.log('test 3');\n  });\n});\n"]},"hash":"99328e08feff4280f9fe60adc81965a2ab43a02a"}
